### ISSUE TITLE
Combo boxes to specify I/O ports missing from tab order of Preferences / Control/OSC web / Add dialog in Windows

### ISSUE THREAD

**Issue by ScottChesworth** [jcsteh/osara#129]
When adding a surface via Preferences / Control/OSC web, the combo boxes where you choose which ports you want each surface to use are missing from the tab order of the Add dialog. They're reachable using object nav or JAWS cursor, but this ux keeps tripping people up, and would be neat to have a fix if you get a free minute to pass it along to Cockos.

Only applies to Windows, VoiceOver navigates the Add dialog just fine.

**Comment by jcsteh** [comment]
I've reported this to Cockos.

**Comment by jcsteh** [comment]
Fixed in REAPER 5.91. Haven't verified myself.

**Comment by ScottChesworth** [comment]
FYI, I have verified. Good times!
